clipper_admin/clipper_admin/deployers/pytorch.py

```diff
@@ -206,28 +206,48 @@ def predict(model, inputs):
         py_minor_version = (sys.version_info.major, sys.version_info.minor)
         # Check if Python 2 or Python 3 image
         if base_image == "default":
-            if py_minor_version < (3, 0):
-                logger.info("Using Python 2 base image")
-                base_image = "{}/pytorch-container:{}".format(
-                    __registry__, __version__)
-            elif py_minor_version == (3, 5):
-                logger.info("Using Python 3.5 base image")
-                base_image = "{}/pytorch35-container:{}".format(
-                    __registry__, __version__)
-            elif py_minor_version == (3, 6):
-                logger.info("Using Python 3.6 base image")
-                base_image = "{}/pytorch36-container:{}".format(
-                    __registry__, __version__)
-            else:
-                msg = (
-                    "PyTorch deployer only supports Python 2.7, 3.5, and 3.6. "
-                    "Detected {major}.{minor}").format(
+            if clipper_conn.cm.gpu:
+                if py_minor_version < (3, 0):
+                    logger.info("Using Python 2 CUDA 10 base image")
```

Comment:
Instead of copying and pasting whole things why don't we create a function for each if statement? Or we even create a class like `RegistryManager`. For example, 
```
if clipper_conn.cm.gpu:
    if py_minor_version < (3, 0):
        base_model = get_base_model(version=2, gpu=True) # and this function logs the info based on the args
```

Reply:
Thanks! This was originally a quick and dirty hack but these are fair points. I'll go ahead and make changes tomorrowish

Reply:
Done